ale_linters/rust/rustc.vim

```diff
@@ -0,0 +1,94 @@
+" Author: Daniel Schemala <istjanichtzufassen@gmail.com>
+" Description: rustc for rust files
+
+if !exists('g:ale_rust_ignore_error_codes')
+    let g:ale_rust_ignore_error_codes = []
+endif
+
+
+function! ale_linters#rust#rustc#handle_rustc_errors(buffer_number, errorlines)
+    let l:file_name = fnamemodify(bufname(a:buffer_number), ':t')
+    let l:output = []
+    for l:errorline in a:errorlines
```

Comment:
Could you add a blank line before any loop or if statement which does not start a new block of indentation?